public/app/plugins/panel/heatmap/HeatmapPanel.tsx

```diff
@@ -218,36 +226,66 @@ export const HeatmapPanel = ({
         {(vizWidth: number, vizHeight: number) => (
           <UPlotChart config={builder} data={facets as any} width={vizWidth} height={vizHeight}>
             {/*children ? children(config, alignedFrame) : null*/}
+            <ZoomPlugin
```

Comment:
are we confident releasing updated ZoomPlugin as part of this (i.e. not behind the FF?)